chrome/content/zotero/components/virtualized-table.jsx

```diff
@@ -811,6 +811,11 @@ class VirtualizedTable extends React.Component {
 		else {
 			return;
 		}
+		// Set aria-activedescendant on table container in case if render() is not called after selection change
+		let selected = this._jsWindow?.getElementByIndex(this.selection.focused);
+		if (selected) {
+			selected.closest(".virtualized-table").setAttribute("aria-activedescendant", selected.id);
+		}
```

Comment:
Ideally something like this would be controlled in a single place in code. Can we remove the code which sets `aria-activedescendant` in `render()` and just do it here, with a comment which says why it's done this way?

Reply:
That makes sense. The only reason why I did not touch that `render` is because when the table is just rendered, `_onSelection` doesn't seem to be called, so that piece in `render` was responsible for setting `aria-activedescendant ` on `collectionTree` the very first time after restart.

But it looks like `_updateTree` of the tree selection is called both during the initial selection and on all further updates after. So I moved `aria-activedescendant` into a separate function and called it in `_updateTree` and it seems to behave as expected. Does that seem reasonable?

I also removed `this.forceUpdate` from selection handlers of item and collection trees because the comments made it look like updating `aria-activedescendant` is the only reason for those calls. Correct me if that's wrong